src/maheshKJun20/ValidateAtmPin.java

```diff
@@ -0,0 +1,79 @@
+package maheshKJun20;
+
+public class ValidateAtmPin {
+	
+	void validatePin(String pin) {
+		if (validateLength(pin)) {
+			if (validateDigit(pin)) {
+				if (validateNumRepitition(pin))
+					if (validateAscendingOrder(pin) || validateDescendingOrder(pin))
+						System.out.println("ATM pin " + pin + " is Valid");
+					else
+						System.out.println("Invalid pin. Order Should be Ascending or Desending");
+			}				
+		}		
+	}
+	
+	boolean validateLength(String pin) {
+		if (pin.length() == 4)
+			return true;
+		else {
+			System.out.println("Invalid pin. Length of pin: " + pin.length());
+			return false;
+		}
+	}
+	
+	boolean validateDigit(String pin) {
+		for (int index=0; index<pin.length();index++) {
```

Comment:
Instead of writing loop you can use try catch --> NumberFormateException
Think about it.